File: 백준/Bronze/1546. 평균/평균.c
#include <stdio.h>

int main() {
    int t, score;
    double sum = 0, max = -1;

    // 시험 본 과목의 개수 입력
    scanf("%d", &t);

    // 과목별 성적 입력 및 최댓값 찾기
    for (int i = 0; i < t; i++) {
        scanf("%d", &score);
        sum += score;
        if (max < score) max = score;
    }

    // 새로운 평균 계산하여 출력
    double average = (sum / max * 100) / t;
    printf("%.2lf\n", average);

    return 0;
}

File: 백준/Bronze/1546. 평균/평균.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf8').trim().split(/\s+/).map(Number);

const N = input[0];

let max = input[1];
for (let i = 2; i <= N; i++) {
  if (input[i] > max) {
      max = input[i];
  }
}

let sum = 0;
for (let i = 1; i <= N; i++) {
  sum += (input[i] / max) * 100;
}

console.log(sum / N);
